nanome_docking/menus/DockingMenu.py

```diff
@@ -105,438 +122,358 @@ def refresh_run_btn_unusable(self, update=True, after=False):
+            self._modes = 9
+
+        if self._modes <= 0:
+            self._modes = 1
+            self._txt2.input_text = self._modes
+            self._plugin.update_content(self._txt2)
 
-        align_btn = menu.root.find_node("AlignButton").get_content()
-        align_btn.register_pressed_callback(align_button_pressed_callback)
-        align_btn.selected = True
+    @async_callback
+    async def slider_released_callback(self, slider):
+        slider.current_value = round(slider.current_value)
+        self._plugin.update_content(slider)
+        self._autobox = slider.current_value
+        self.size_value_txt.text_value = str(slider.current_value)
+
+        if hasattr(self, 'site_sphere'):
+            # Update site sphere to show new radius
+            self.site_sphere.radius = slider.current_value
+            await Shape.upload(self.site_sphere)
+        self._plugin.update_content(self.size_value_txt)
+
+    def pose_added_callback(self, button):
+        self._modes += 1
+        self._txt2.input_text = self._modes
+        self._plugin.update_content(self._txt2)
+
+    def pose_subbed_callback(self, button):
+        self._modes -= 1
+        if self._modes <= 0:
+            self._modes = 1
+        self._txt2.input_text = self._modes
+        self._plugin.update_content(self._txt2)
+
+    @async_callback
+    async def loc_refresh_pressed_callback(self, button):
+
+        if not self._selected_site:
+            Logs.debug("No Site Selected")
+            self._site_x.input_text, self._site_y.input_text, self._site_z.input_text = '', '', ''
+            self._plugin.update_menu(self._menu)
+        else:
+            Logs.debug("Update the site location")
+            comp = self._selected_site.complex
+            comp = (await self._plugin.request_complexes([comp.index]))[0]
+            self._site_x.input_text, self._site_y.input_text, self._site_z.input_text = [round(x, 2) for x in comp.position]
+            radius = self._slider.current_value
+            asyncio.create_task(self.draw_site_sphere(comp, radius))
+            self._plugin.update_menu(self._menu)
 
-        self._score_btn = menu.root.find_node("ScoringButton").get_content()
-        self._score_btn.register_pressed_callback(scoring_button_pressed_callback)
+    @property
+    def dd_ligands(self):
+        return self._menu.root.find_node("LigandDropdown").get_content()
 
-        self._display_score_btn = setting_menu.root.find_node("VisualScoresButton").get_content()
-        self._display_score_btn.register_pressed_callback(visual_scores_button_pressed_callback)
+    @property
+    def dd_receptor(self):
+        return self._menu.root.find_node("ReceptorDropdown").get_content()
 
-        close_score_btn = menu.root.find_node("CloseScoreButton").get_content()
-        close_score_btn.register_pressed_callback(close_score_pressed_callback)
+    @property
+    def dd_site(self):
+        return self._menu.root.find_node("SiteDropdown").get_content()
```

Comment:
Why the `@property`s here instead of `self.dd_... = ...` in `build_menu`?

Reply:
I've been using properties for accessing content in other plugins, and I was gonna refactor this one to match, then I didn't haha. I'll remove these ones and keep it consistent within the plugin.